aeon/dj_pipeline/analysis/visit_analysis.py

```diff
@@ -366,7 +368,7 @@ def make(self, key):
                         **food_patch_key,
                         "visit_date": visit_date.date(),
                         "time_fraction_in_patch": in_patch.mean(),
-                        "in_patch": in_patch.values,
+                        "in_patch": np.array(in_patch.index[np.where(in_patch)]),
```

Comment:
```suggestion
                        "in_patch": np.array(in_patch.index[in_patch]),
```
would this work instead?